kyuubi-common/pom.xml

```diff
@@ -52,6 +52,11 @@
             <artifactId>slf4j-log4j12</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+        </dependency>
```

Comment:
test only

Reply:
kyuubi-server's test suites can not see this dependency if scope is set to test.

Reply:
kyuubi-server's test suites can not see this dependency if scope is set to test.

Reply:
Add it in `kyuubi-server/pom.xml` again

Reply:
> Add it in kyuubi-server/pom.xml again
This is quite inconvenient because all modules depends `kyuubi-common` in test scope have to add `jul-to-slf4j`.